stylus/base/helpers.styl

```diff
@@ -116,49 +116,53 @@ for $name, $shade in $shades
   .has-background-{$name}
     background-color: $shade !important
 
-.has-text-weight-light
-  font-weight: $weight-light !important
-.has-text-weight-normal
-  font-weight: $weight-normal !important
-.has-text-weight-semibold
-  font-weight: $weight-semibold !important
-.has-text-weight-bold
-  font-weight: $weight-bold !important
+.has-text-weight
```

Comment:
Much clean now 👍

btw, is it a good idea if we use `for-loop` to generate these helpers?
It'll looks like:

```stylus
$weights = 'light' 'normal' 'medium' 'semibold' 'bold'

for $weight in $weights
  .has-text-weight-{$weight}
    font-weight: lookup('$weight-'+$weight) !important
```

If it looks good, may I please you help to change them (๑ơ ω ơ)？

Reply:
Sure. It's done.